IBM/MAX-Sports-Video-Classifier
# test other endpoints

- test other endpoints
- install from requirements.txt (resolves #8)
- check md5 hashes of downloaded files
- Add request param descriptions and update Swagger models help text (resolves #6)
